Source/Tooltips/Main.lua

```diff
@@ -20,61 +20,53 @@ function Auctionator.Tooltip.ShowTipWithPricing(tooltipFrame, itemId, itemCount)
     countString = "|cFFAAAAFF x" .. itemCount .. "|r"
   end
 
-  local auctionPrice = Auctionator.Database.GetPrice(itemId)
+  local auctionPrice = Auctionator.Database.GetPrice(itemKey)
   if auctionPrice ~= nil then
     auctionPrice = auctionPrice * (showStackPrices and itemCount or 1)
   end
+  
+  local vendorPrice = 0;
+  local cannotAuction = 0;
 
-  local
-    name,
-    link,
-    rarity,
-    level,
-    minLevel,
-    itemType,
-    itemSubType,
-    stackCount,
-    equipLocation,
-    icon,
-    sellPrice,
-    classID,
-    unused,
-    cannotAuction = GetItemInfo(itemId);
-
-  -- TODO Listen to GET_ITEM_INFO_RECEIVED to get info for non-cached items.
-  if name~=nil then
-    local vendorPrice = sellPrice * (showStackPrices and itemCount or 1)
-
-    tooltipFrame:AddDoubleLine("ItemID", itemId)
-
-    Auctionator.Tooltip.AddVendorTip(tooltipFrame, vendorPrice, countString)
-    Auctionator.Tooltip.AddAuctionTip(tooltipFrame, auctionPrice, countString, cannotAuction)
+  if string.sub(itemKey,1,1) == "p" then
```

Comment:
I think we want a utility method for this as well:
```lua
if Auctionator.Utilities.IsPetItemKey(itemKey) then
  -- etc.
end
```

Reply:
Will do